controllers/operator/nmstate_controller.go

```diff
@@ -389,3 +439,19 @@ func setClusterReaderExist(c client.Client, data render.RenderData) error {
 	data.Data["ClusterReaderExists"] = found
 	return nil
 }
+
+func getEnv(key, fallback string) string {
```

Comment:
There is a `pkg/environment/environment` module for this, can you move there and rename to follow the pattern ?

Reply:
moved that function to the `environment` package